### PR TITLE
Relate items carousel behaviour

### PR DESCRIPTION
**Pull request by Gledinburgh**
some minor behaviour changes along with accessibility and styling improvements .

### CODE REVIEW

File: client/dist/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (rick-o-H) [inline client/dist/index.html:2]: NICE

Comment (rick-o-H) [reply client/dist/index.html:2]: I remember reading somewhere that including this helps! not sure why

File: client/src/RealatedComparisonComponents/ComparisonModal.jsx [hunk 2]
```diff
@@ -27,7 +27,11 @@ class ComparisonModal extends React.Component {
   remapFeaturesInfo (rawProductFeatures) {
     var mappedFeatures = {};
     rawProductFeatures.forEach((featureObj) => {
-      mappedFeatures[featureObj.feature] = featureObj.value;
+      if (featureObj.value === 'null') {
```
Comment (rick-o-H) [inline client/src/RealatedComparisonComponents/ComparisonModal.jsx:30]: Does this cover _null_ and 'null'? I found a case today in my module where they were throwing me null and 'null' and it broke my component lol

**Review by rick-o-H — APPROVED** [review]
Looks great @Gledinburgh !

File: client/src/RealatedComparisonComponents/ComparisonModal.jsx (hunk 2 above)
Comment (Gledinburgh) [reply client/src/RealatedComparisonComponents/ComparisonModal.jsx:30]: no harm in doing both I suppose.

**Pull request merged**